Common/src/main/resources/assets/biomeswevegone/lang/ru_ru.json

```
@@ -101,12 +113,13 @@
   "block.biomeswevegone.aspen_stairs": "Осиновые ступени",
```

Все "ступени" в этом файле должны стать "ступеньками" как в офф. переводе.
```suggestion
  "block.biomeswevegone.aspen_stairs": "Осиновые ступеньки",
```